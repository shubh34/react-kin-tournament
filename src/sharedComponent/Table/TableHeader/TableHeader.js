// import './TabsContent.scss';
import React from 'react';
import PropTypes from 'prop-types';

const TableHeader = ({ children }) => (
	<th>
		{children}
	</th>
);


TableHeader.propTypes = {
	children: PropTypes.node.isRequired,
};

export default TableHeader;
