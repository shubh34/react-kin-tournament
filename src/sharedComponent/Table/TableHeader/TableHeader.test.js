import React from 'react';
import { shallow } from 'enzyme';
import TableHeader from './TableHeader';

describe('TableHeader', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<TableHeader> Table Header </TableHeader>);
		expect(wrapper).toMatchSnapshot();
	});
});
