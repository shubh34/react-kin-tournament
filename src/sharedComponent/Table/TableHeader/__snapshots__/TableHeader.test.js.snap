// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader renders without crashing 1`] = `
<th>
   Table Header 
</th>
`;
